pipelines/radiosonde/config/dataset.yaml

```diff
@@ -0,0 +1,245 @@
+attrs:
+  title: Radiosonde
+  description: Taking data for Radiosonde data from CA OAKLAND/METROP. OAKLAND INT. and converting it from ascii text to nc.
```

Comment:
Please add a description of the measurements, not what the pipeline does. You should be able to copy this from the original file.